  public void testPromotInteger() throws IOException {
    runTest(INTEGERS, TestType.Int);
  }

